blocks/loader.ts

```diff
@@ -1,7 +1,7 @@
 // deno-lint-ignore-file no-explicit-any
-import hash from "https://esm.sh/v135/object-hash@3.0.0";
 import JsonViewer from "../components/JsonViewer.tsx";
 import { ValueType } from "../deps.ts";
+import crypto from "node:crypto";
```

Comment:
you can use crypto without importing from node

Reply:
actually deno has it's own crypto implementation

https://deno.land/api@v1.39.4?s=Crypto

Reply:
but this module is just to create randoms

Reply:
it is not stable, as my current implementation, so I'm just optimizing the previous approach